stdlib/public/core/StringComparison.swift

```diff
@@ -298,7 +294,7 @@ internal func _lexicographicalCompare(
   let rightHSPtr = rightHS.baseAddress._unsafelyUnwrappedUnchecked
   return _lexicographicalCompare(leftHSPtr[idx], rightHSPtr[idx])
 }
-// @inline(never) // TODO: temporary, to inspect code quality
+@inline(__always)
```

Comment:
Why do you think this should always be inlined?

Reply:
This matches the other lexicographicalCompare overloads in this file